<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sproutClass</class>
 <widget class="QMainWindow" name="sproutClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>473</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>464</width>
    <height>473</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>464</width>
    <height>473</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>2Sprout</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/2sprout.png</normaloff>:/icons/2sprout.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>464</width>
     <height>473</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>464</width>
     <height>473</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>471</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>451</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string>2sprout Status</string>
      </property>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>29</y>
         <width>431</width>
         <height>381</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit { 
border-width: 5px; 
border-style: inset; 
border-color: #535353; 
border-radius: 9px; 
background: #ffffff; 
color: #000000; 
}</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>440</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/started.png</normaloff>icons/started.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>440</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/stop.png</normaloff>icons/stop.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Advanced</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/sprites_09.png</normaloff>icons/sprites_09.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>9</y>
        <width>431</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Database</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Select Database:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Server Host:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Default DB:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>70</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>161</width>
         <height>26</height>
        </rect>
       </property>
       <property name="minimumContentsLength">
        <number>0</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>PostgreSQL</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/db_add.png</normaloff>:/icons/db_add.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MySQL</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/db_add.png</normaloff>:/icons/db_add.png</iconset>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>localhost</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>70</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>5432</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>130</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>160</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>440</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Test</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/db_host.png</normaloff>icons/db_host.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/sprites_09.png</normaloff>icons/sprites_09.png</iconset>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>431</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Networking</string>
      </property>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>281</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable automatic router port forwarding</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>4950</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>431</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>DB Connection Status</string>
      </property>
      <widget class="QTextEdit" name="textEdit_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>411</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit { 
border-width: 3px; 
border-style: inset; 
border-color: #535353; 
border-radius: 9px; 
background: #ffffff; 
color: #000000; 
}</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
